ocs_sample_library_preview/DataViews.py

```diff
@@ -304,6 +304,73 @@ def getDataInterpolated(self, namespace_id: str = None, data_view_id: str = None
             return content, nextPage, firstPage
         return value_class.fromJson(content), nextPage, firstPage
 
+    def getDataStored(self, namespace_id: str = None, data_view_id: str = None,
```

Comment:
We did this a little differently in `Streams.py`. In that client, there is a separate function for passing in the `url` directly, that would be like `getDataStoredUrl`. Then `getDataStored` would accept a `namespace_id` and `data_view_id` to format the `url` and pass it (and all the query parameters) to `getDataStoredUrl`. See [here](https://github.com/osisoft/sample-ocs-sample_libraries-python/blob/main/ocs_sample_library_preview/Streams.py#L557) and [here](https://github.com/osisoft/sample-ocs-sample_libraries-python/blob/main/ocs_sample_library_preview/Streams.py#L579). 
Generally, if we'd never had the `url` parameter before, I'd say this is fine, but we should probably use a consistent pattern for it.

Reply:
That makes sense, but I did it the way that matches the existing data views interpolated data call: [here](https://github.com/osisoft/sample-ocs-sample_libraries-python/blob/30d0442c8c50269e202eb6bfa620db42df45dc7f/ocs_sample_library_preview/DataViews.py#L238)

I can add an interpolation url function to have both match the streams style, but I can't change the existing interpolation function to remove the url handling because I know there are customers using it to page. What I could do though is alias the URL handling portion of it to just call the newly created URL function. 

Thoughts on that?

Reply:
Ah, I see what you mean now. I suppose we should just keep it consistent with the way the data views client does it already. Probably no reason to support doing the url calls two separate ways in the DV client.